setup_cosim_build_env.sh

```diff
@@ -17,6 +17,7 @@ export USE_DISTRIBUTED=0
 export USE_OPENMP=0
 export ATEN_THREADING=NATIVE
 export CFLAGS='-fuse-ld=gold'
+export OMP_NUM_THREADS=1
```

Comment:
USE_OPENMP=0?

Reply:
Somehow pytorch also uses this `OMP_NUM_THREADS` to control its pthread pool.
https://github.com/pytorch/pytorch/issues/975#issuecomment-396392836
just playing safe. because when i was training on CPU, I did see a lot of worker threads